File: app/src/main/java/com/example/blockforce/ScoreDatabaseHelper.java
package com.example.blockforce;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;


public class ScoreDatabaseHelper extends SQLiteOpenHelper {
    static final private String DBNAME = "score.sqlite";
    static final private int VERSION = 1;
    public ScoreDatabaseHelper(Context context) {
        super(context, DBNAME, null, VERSION);
    }

    @Override
    public void onOpen(SQLiteDatabase db) {
        super.onOpen(db);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL("CREATE TABLE score (" + "Lines INTEGER PRIMARY KEY, date TEXT,Latest INTEGER)");
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }
}

File: app/src/main/java/com/example/blockforce_kotlin/ScoreDatabaseHelper.kt
package com.example.blockforce_kotlin

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class ScoreDatabaseHelper(context: Context) : SQLiteOpenHelper(context, DBNAME, null, VERSION) {
    override fun onOpen(db: SQLiteDatabase?) {
        super.onOpen(db)
    }

    override fun onCreate(db: SQLiteDatabase) {
        db.execSQL("CREATE TABLE score (" + "Lines INTEGER PRIMARY KEY, date TEXT,Latest INTEGER)")
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {}

    companion object {
        private val DBNAME: String = "score.sqlite"
        private const val VERSION = 1
    }
}